modules/tinymce/src/plugins/table/main/ts/api/Settings.ts

```diff
@@ -60,6 +62,7 @@ const isPixelsForced = (editor: Editor): boolean => getTableSizingMode(editor) =
 const isResponsiveForced = (editor: Editor): boolean => getTableSizingMode(editor) === 'responsive';
 const getToolbar = (editor: Editor): string => editor.getParam('table_toolbar', defaultTableToolbar);
 
+const getColumnGroupUsage = (editor: Editor): boolean => editor.getParam('table_col_group', columnGroupDefault, 'boolean');
```

Comment:
Maybe `table_use_colgroups` instead? At the very least there shouldn't be an underscore between `col_group`, unless spelling out the full name.

Reply:
Also, this function isn't really a getter as it's a returning a boolean, so it should start with `is` or something similar instead. Maybe `useColumnGroups`?